installer/dev/install.h

```diff
@@ -13,5 +26,11 @@ namespace WindowsAppRuntimeInstaller {
         bool isFramework{};
     };
 
-    HRESULT DeployPackages(const bool quiet) noexcept;
+    HRESULT Deploy(const WindowsAppRuntimeInstaller::Options options) noexcept;
+
+    HRESULT DeployPackages(const WindowsAppRuntimeInstaller::Options options);
```

Comment:
Code seems like an odd mix of exceptional and HRESULTish error patterns.

Reply:
Pre-existing condition.

There's some odd mixes as you say, but we'll bake those out as we do more work in this space next year. I made a few minor improvements as needed to make (and test!) this PR but didn't want to range too far afield into other code paths as we're prepping for release.

We'll get there, just not today.